spec/factories/users.rb

```diff
@@ -11,6 +11,12 @@
     factory :valid_princeton_patron do
     end
 
+    factory :campus_authorized_patron do
```

Comment:
@kevinreiss I do not think these are needed.

Reply:
Updated